projects/wave-core-new/src/lib/operators/dialogs/histogram-operator/histogram-operator.component.ts

```diff
@@ -83,7 +83,7 @@ export class HistogramOperatorComponent implements OnInit, AfterViewInit, OnDest
                 },
             ),
             autoBuckets: [true, Validators.required],
-            numberOfBuckets: [20, Validators.required],
+            numberOfBuckets: [100, [Validators.required, Validators.min(0), Validators.max(100)]],
```

Comment:
* the default should stay at 20
* I guess we need at least one bucket as minimum

Reply:
1f0a88cacaa268afe16677b783518a944777aee1